src/edge_index.cpp

```diff
@@ -135,32 +130,33 @@ std::unordered_map<NodeID, RawIndex> _groupNodeRanges(const std::vector<NodeID>&
 }
 
 
-std::vector<NodeID> _readNodeIDs(const HighFive::Group& h5Root, const std::string& name)
-{
+std::vector<NodeID> _readNodeIDs(const HighFive::Group& h5Root, const std::string& name) {
     std::vector<NodeID> result;
     h5Root.getDataSet(name).read(result);
     return result;
 }
 
 
-void _writeIndexDataset(const RawIndex& data, const std::string& name, HighFive::Group& h5Group)
-{
+void _writeIndexDataset(const RawIndex& data, const std::string& name, HighFive::Group& h5Group) {
     auto dset = h5Group.createDataSet<uint64_t>(name, HighFive::DataSpace::From(data));
     dset.write(data);
 }
 
 
-void _writeIndexGroup(const std::vector<NodeID>& nodeIDs, uint64_t nodeCount, HighFive::Group& h5Root, const std::string& name)
-{
+void _writeIndexGroup(const std::vector<NodeID>& nodeIDs,
+                      uint64_t nodeCount,
+                      HighFive::Group& h5Root,
+                      const std::string& name) {
     auto indexGroup = h5Root.createGroup(name);
 
     auto nodeToRanges = _groupNodeRanges(nodeIDs);
-    const auto rangeCount = std::accumulate(
-        nodeToRanges.begin(), nodeToRanges.end(), uint64_t(0),
-        [](uint64_t total, decltype(nodeToRanges)::const_reference item) {
-            return total + item.second.size();
-        }
-    );
+    const auto rangeCount =
+        std::accumulate(nodeToRanges.begin(),
+                        nodeToRanges.end(),
+                        uint64_t(0),
+                        [](uint64_t total, decltype(nodeToRanges)::const_reference item) {
```

Comment:
I guess you are using C++14
```suggestion
                        [](uint64_t total, const auto& item) {
```

Reply:
Yes, however this is not from the reports part (it comes from upstream).
The changes are due to clang-format.